%YAML:1.0
BiasAcc: 1.0e-06
BiasGyr: 1.0e-06
NeighborNum: 0.0
NeighborRange: -1.0
NeighborStd: 0.0
PointUpdateTh: 0.98
Pos: 1.25e-09
PosExt: 1.0e-08
RollPitch: 1.0e-06
RotExt: 1.0e-08
Vel: 0.001
Yaw: 1.0e-06
azimuthCov: 0.5
capacity: 5500
dop_outlier_th: 0.2
dopplerCov: 0.1
elevationCov: 10.0
eps: 0.0001
imuAccBias: 7.237989999999999e-06
imuAccNoise: 0.0060485999999999995
imuGravity: 9.796
imuGyrBias: 0.0004616800000000003
imuGyrNoise: 0.0003026679999999999
imuInitVel: 0.01
imu_topic: /gx5/imu/data
kf_angle_deg: 5
kf_distance: 0.3
kf_frame: 10
maxDoppler: 0.025
maxRange: 15.0
max_iteration: 10
minIntensity: 0.2
minRange: 0.5
radar_topic: /mmWaveDataHdl/RScan
rangeCov: 0.05
res_outlier_th: 0.5
voxel_size: 0.6

T_Radar_to_Imu: !!opencv-matrix
   rows: 4
   cols: 4
   dt: d
   data: [ 0.99999995, 0.00079265,  0.00059265,  0.02995821,
           0.00079265, -0.99999969,  0.00000000, 0.12002374,
          -0.00059265, 0.00000047, -0.99999982,  -0.08998226,
           0.00000000, 0.00000000,  0.00000000,  1.00000000]